pkg/apis/core/types.go

```diff
@@ -2654,6 +2679,11 @@ type PodStatus struct {
 	// when we have done this.
 	// +optional
 	ContainerStatuses []ContainerStatus
+
+	// List of user-initiated ephemeral containers that have been run in this pod. This list is append-only.
+	// This field is alpha-level and is only honored by servers that enable the DebugContainers feature.
```

Comment:
EphemeralContainers feature?

Reply:
The DebugContainers gate was added quite a while ago. It's still known to users as Debug Containers, but given that the gate is for the apiserver & kubelet it makes sense to rename it. I will do so in this PR since the same approvers can likely also approve the feature rename.